test/intl402/Segmenter/constructor/supportedLocalesOf/result-type.js

```diff
@@ -21,14 +21,13 @@ assert.sameValue(Object.isExtensible(result), true,
 assert.notSameValue(result.length, 0);
 for (let i = 0; i < result.length; ++i) {
   verifyProperty(result, String(i), {
-    "writable": false,
+    "writable": true,
     "enumerable": true,
-    "configurable": false,
+    "configurable": true,
   });
 }
 
 verifyProperty(result, "length", {
-  "writable": false,
```

Comment:
The verifyProperty function is pretty buggy now. IF you can fix it, I can added it back. According to jakobkummerow 3 hours ago
> It looks like verifyProperty is too limited to successfully check writability of Array.length, so I'd just drop this line.

Reply:
Would you explain what's exactly the bug here or how is it limited?

Reply:
https://tc39.github.io/ecma262/#sec-property-attributes said
"
[[Writable]] | Boolean | If false, attempts by ECMAScript code to change the property's [[Value]] attribute using [[Set]] will not succeed.
"
Notice it does not stated "throw TypeError" but "not succeed". I somehow got a "RangeError" (which is "not succeed") but not TypeError. 

But in isWritable the function inside harness/propertyHelper.js which check the writable or not, it is coded specifically checking for TypeError and won't consider other kind of "not succeded".  Even if I change to include RangeError, I also hit some issue (I cannot figure out what exactly) with "length".

Reply:
@ jakobkummerow

Reply:
When a `"writable": true` property is in the options object, then `verifyProperty` [calls](https://github.com/tc39/test262/blob/master/harness/propertyHelper.js#L64) `isWritable(array, "length")`, which in absense of the optional 4th argument will [try](https://github.com/tc39/test262/blob/master/harness/propertyHelper.js#L129) to set `array["length"] = "unlikelyValue"`, which throws a `RangeError`, which causes `isWritable` to throw an exception because it will only accept TypeErrors there. Even if that is fixed, it will then [check](https://github.com/tc39/test262/blob/master/harness/propertyHelper.js#L136) whether `array.length === "unlikelyValue"` after the assignment, which will of course never be true, even though the property is, in fact, writable.

In short, `verifyProperty` in its current form cannot be used to assert writability of `Array.length`, and it is clearly beyond the scope of this patch to fix that.

A workaround would be to do a separate `verifyWritable(result, "length")` call after the writable-ignoring `verifyProperty` call.

Reply:
The difficulty of using the property verification methods has been raised by many different test262 contributors. Maybe the testing should be based on getOwnPropertyDescriptor by default, and the more intense testing can be used in cases where we're especially suspicious of a potential issue (e.g., in RegExp legacy properties).

Reply:
By the reasons pointed by @jakobkummerow, the writable check doesn't work for `Array#length` (thanks for clarifying that, btw). Although, a verifyWritable call would have the same result and issue.

In this case it would be nice to just have a simple check for the writable attr in the descriptor of this case. We might discuss other issues elsewhere not blocking this PR.